_translator-files/po/zh_CN/Running-a-Server.po

```diff
@@ -173,16 +176,20 @@ msgid ""
 "<figcaption>How Registered Servers work</figcaption>\n"
 "</figure>\n"
 msgstr ""
+"<figure>\n"
+"\t<img src=\"{% include img/en-screenshots/diagram-reg-server.inc %}\" loading=\"lazy\" alt=\"Diagram of connections between Clients within a Jamulus Registered Server\">\n"
+"<figcaption>How Registered Servers work</figcaption>\n"
```

Comment:
Preferably, the figcaption text should be translated.

Reply:
done